### PR TITLE
refactor: 💡 use const for signer in tests

### PR DESCRIPTION
**Pull request by polymath-eric**
### JIRA Link 

Prep work for: [DA-375](https://polymesh.atlassian.net/browse/DA-375)

### Changelog / Description 

Refactors repeated '0x06'.padEnd calls into a `const`. Also separates signer and DID in some tests

### Checklist - 

- [ ] New Feature ?
- [ ] Updated swagger annotation (if API structure is changed) ?
- [ ] Unit Test (if possible) ?
- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

**Comment by polymath-eric** [comment]
I think our document examples could get a similar treatment, but I'll save that for another PR.

File: src/assets/assets.controller.spec.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ import { createAuthorizationRequestModel } from '~/authorizations/authorizations
 import { PaginatedResultsModel } from '~/common/models/paginated-results.model';
 import { ComplianceRequirementsService } from '~/compliance/compliance-requirements.service';
 import { PortfolioDto } from '~/portfolios/dto/portfolio.dto';
+import { testDid } from '~/test-utils/consts';
```
[the remark this answers is not in the document]
Comment (polymath-eric) [reply src/assets/assets.controller.spec.ts:17]: I was thinking about that when I was writing it.

My reasoning is signer is in every request, where as DID is more request specific.

**Comment by polymath-eric** [comment]
> also noticed there are a lot more places where we define did & sinner 

My intention is to keep these refactors small and relatively quick to review.

We use signer and did extensively in the doc comments, but I think thats another concern and we shouldn't recycle values for them. Instead we should make a documentation consts file and use those.

**Review by prashantasdeveloper — APPROVED** [review]

**Pull request merged**